changelog/changelog2022.md

```diff
@@ -0,0 +1,11 @@
+# Changelog 2022
+
+## 19th September 2022
+
+* [Sources](../mews-operations/reservations.md#source) replaces Channel in the [Process Group Request](../mews-operations/reservations.md#request), which now supports processing of multiple sources.
```

Comment:
I would say "In the [Process Group](../mews-operations/reservations.md#process-group) request message, Channel is deprecated and replaced by Sources. Sources is a collection of new object [Source](../mews-operations/reservations.md#source), which provides additional information about reservation sources or channels."

Reply:
Actually, Channel, Sources and Source should not be capitalized, so better to write them as `channel`, `sources` and `source`, I think that's the convention that's been used before to refer to fields or properties within messages.